package.json

```diff
@@ -39,7 +39,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --update-snapshot",
+    "test": "react-scripts test --update-snapshot --coverage --watchAll",
```

Comment:
I think the watch all you added did not make the test to terminate after finishing with the test. That's why Travis is failimg

Reply:
done. thanks